cli/tests/activate.bats

```diff
@@ -114,13 +114,24 @@ project_teardown() {
 
 # Wait for a backgrounded activation to start, signalled by a blocking writing
 # to a FIFO file.
+# $1: background_pid
+#       This PID must be in a new process group otherwise this will kill bats as
+#       well
```

Comment:
I don't love this but don't have any other ideas at the moment. I'm not sure if there would be a more straightforward way to tell Bash to cleanup after itself

Reply:
Is this another case where sandboxing would help?